providers/base/units/rtc/jobs.pxu

```diff
@@ -14,3 +14,91 @@ user: root
 category_id: rtc_category
 estimated_duration: 40
 flags: noreturn
+
+id: rtc/rtc_number
+_summary: Check the number of RTC as expected.
+_description:
+    Check the number of RTC on platform as expected.
+plugin: shell
+user: root
+estimated_duration: 1.0
+category_id: com.canonical.certification::rtc_category
+command:
+    if [ -z "$TOTAL_RTC_NUM" ]; then 
+        TOTAL_RTC_NUM=1; 
+    fi
+    SYSFS_RTC_NUM=$(find /dev/ -name "rtc[0-9]" | wc -l)
+    echo "TOTAL_RTC_NUM: $TOTAL_RTC_NUM"
+    echo "Number of RTC devices found in sysfs: $SYSFS_RTC_NUM"
+    if [ "$SYSFS_RTC_NUM" != "$TOTAL_RTC_NUM" ]; then
+        echo "RTC number mismatch"
+        exit 1
+    else
+        echo "RTC number matched"
+    fi
+flags: also-after-suspend
+
+id: rtc_list
+_summary: Generates a rtc list for the platform
+_description:
+ List out all rtc devices on platform, and generate a list for it.
+ Define TOTAL_RTC_NUM in config file if platform have more then one
+ RTC deivces.
+plugin: resource
+user: root
+estimated_duration: 1.0
+category_id: com.canonical.certification::rtc_category
+command:
+    rtcs=$(find /dev/ -name "rtc[0-9]"| awk -F "/" '{print $3}')
+    for c in ${rtcs}; do
+        echo "rtc: ${c}"
+        echo ""
+    done
+
+unit: template
+template-resource: rtc_list
+template-unit: job
+id: rtc/rtc_alarm_{rtc}
+_summary: RTC alarm of {rtc} is worked
+_purpose:
+    Check if RTC alarm of {rtc}
```

Comment:
And if you change the summary to what I suggest, you may not need this, unless you want to use `_purpose` section to provide even more details.